orchagent/main.cpp

```diff
@@ -143,14 +160,12 @@ int main(int argc, char **argv)
     int opt;
     sai_status_t status;
 
-    bool disableRecord = false;
-
     while ((opt = getopt(argc, argv, "m:hR")) != -1)
```

Comment:
we need to be able to specify the directory of those log files. this is because we need to mount a directory so that we do not store those record files in the docker, but outside the docker, so that we can manage the disk space of it.